tests/formula/program-formula.txt

```diff
@@ -30,6 +31,7 @@ A022322: a(n) = b(n-2)+1, a(1) = 8, a(0) = 1, b(n) = b(n-2)+b(n-1)+2, b(1) = 18,
 A022958: a(n) = 2-n
 A037536: a(n) = (9*b(n-2))/13, a(1) = 5, a(0) = 1, b(n) = 3*b(n-1), b(0) = 24
 A048745: a(n) = 2*a(n-1)+a(n-2)+3, a(1) = 5, a(0) = 1
+A062815: a(n) = (n+1)*(n+1)^(n+1)+a(n-1), a(0) = 1
 A112032: a(n) = 2*a(n-2), a(1) = 1, a(0) = 4
```

Comment:
Sort the initial states, so that `a(0)` comes before `a(1)`.

Reply:
If I revert the order, it would be this: `A112032: a(0) = 4, a(1) = 1, a(n) = 2*a(n-2)`. I prefer having the recurrence in the front.

Reply:
It's regarding the `A048745: a(n) = 2*a(n-1)+a(n-2)+3, a(1) = 5, a(0) = 1`.

Here it would look nicer with sorted initial values, like this: `A048745: a(n) = 2*a(n-1)+a(n-2)+3, a(0) = 1, a(1) = 5`